drivers/isl29125/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = isl29125
```

Comment:
I think as long as the module and directory are the same it can be omitted.. will test.